lib/src/block_parser.dart

```diff
@@ -114,10 +126,21 @@ class BlockParser {
   BlockSyntax? get parentSyntax => _parentSyntax;
   BlockSyntax? _parentSyntax;
 
+  /// If the [SetextHeadingSyntax] is disabled temporarily
+  bool get setextHeadingDisabled => _setextHeadingDisabled;
+  bool _setextHeadingDisabled = false;
+
+  /// The [BlockSyntax] which is running now.
```

Comment:
Could you note in this comment when `currentSyntax` is `null`.

Reply:
Fixed